### ISSUE TITLE
Structmapping needs more specific headers for individual sections

### ISSUE THREAD

**Issue by lightgivener** [pulibrary/cicognara-catalogo#122]
Hey!

This is a good example of where we need more specific structmapping titles. That I cannot do myself, correct?

The problem here is that Cicognaras Catalogo entry combines such a range of publication years under one Cico No entry. Many of them are double numbers to. 

https://figgy.princeton.edu/catalog/e853e547-16c7-4933-82fe-a41b784ddd5c

**Comment by smallfind** [comment]
Right. I believe that Esmé has shown me how to do this a loooonnnnng time ago but I don't remember how to. @escowles and @tpendragon -- Can you remember/ give us an idiot's guide to doing this?

**Comment by tpendragon** [comment]
There's a bug here I didn't know about. It should be that if you remove the source metadata identifier and then set the title it should use that title, but it doesn't work. Please see https://github.com/pulibrary/figgy/issues/3818. I've attached the issue to our current sprint, so hopefully we can get this dealt with soon.

**Comment by tpendragon** [comment]
@smallfind You can do this now. To do it you just go into each volume, delete the source metadata identifier, and change the title to whatever you want. Probably "Volume 1" or something similar?

**Comment by lightgivener** [comment]
Hey! Great info, thanks. I just tried this now and got a 'sorry something went wrong' error message - but it could also be that I just don't have the rights to do this. Cheers!

**Comment by smallfind** [comment]
@lightgivener : Can you suggest appropriate titles for the various volumes? I'll try to add them.

**Comment by lightgivener** [comment]
@smallfind hey! for this I would suggest just adding the year of publication (from the title page)? Just something that helps a user navigate and identify what they are looking for and that is not just the same title for all. In other examples it would be a volume designator or for if the titles vary much like in the Trattato and accompanying Disegno example we had here yesterday. Does that make sense? I am not fixed on any particular solution here, just something that makes one more aware what might hide between the different structmap sections.

cheers!
